### ISSUE TITLE
unable to unset advanced owl properties

### ISSUE THREAD

**Issue by alexskr** [ncbo/bioportal-project#283]
when editing advanced owl properties in the UI it is not possible to unset a property which was been set previously.

I am able to delete contents of the input box and click on the save button but those properties re-appear when I refresh that page.

**Comment by jonquet** [comment]
We have identified this and @syphax-bouazzouni shall be able to make a link

**Comment by syphax-bouazzouni** [comment]
> We have identified this and @syphax-bouazzouni shall be able to make a link

I guess you are speaking about this one https://github.com/agroportal/project-management/issues/420 and this https://github.com/agroportal/project-management/issues/417

**Comment by graybeal** [comment]
Yes, though the second one is a bit ambiguous. The issues Alex saw were the first one (#420), and then we saw (for the author field) the pre-processing failed miserable, so the ontology could no longer be processed for somewhat opaque reasons.  There are other fields that would probably have the same impact.
